### ISSUE TITLE
Handle Traffic Timeout Performance Hit when the ClojureDocs cache is missing

### ISSUE THREAD

**Issue by tim14smith** [clojure-emacs/orchard#85]
Related to #71 

##  Expected behavior

CIDER stacktraces should return relatively quickly even when ClojureDocs cache is missing and all internet traffic is set to time out.

## Actual behavior

CIDER stacktrace takes 1 second per special form due to the 1-second timeout being repeated for every call to `see-also` attempting to load the cache file.

## Steps to reproduce the problem

Set all internet traffic to drop. 

If using a VPN, look for a killswitch setting or ability to stop all traffic when not connected and then disconnect from VPN. (Some VPNs with killswitches: https://www.vpnmentor.com/blog/kill-switch-features/)

If using `iptables`:

```sh
# NOTE: this will drop all internet traffic. To re-enable, use ACCEPT in place of DROP
sudo iptables -F
sudo iptables -P INPUT DROP
sudo iptables -P OUTPUT DROP
sudo iptables -P FORWARD DROP
```

Delete `orchard` subdirectory from the cache directory (as specified in https://github.com/clojure-emacs/orchard/blob/master/src/orchard/util/os.clj#L58).

Open a CIDER nREPL in any Clojure project and execute code that spawns an error. This was tested in Emacs with the current version of cider.

```clojure
;; Error shows immediately in REPL window, but took ~15 seconds to show error buffer in Emacs
(/ 1 0)
```

## Environment & Version information

Orchard 0.5.7, Cider 20200308.1030

### Clojure version

Tested with 1.10.1

### Java version

Tested with Java 8 and 11

### Operating system

Tested on Linux and MacOS

**Comment by fdhenard** [comment]
Hey @tim14smith, thanks for reporting this!

Have you figured out any workarounds for it?

**Comment by bbatsov** [comment]
In hindsight I'm starting to wonder if it was a good idea to try to dynamically update this instead of just bundling a snapshot of data and updating it between releases. On one hand the cache update clearly causes issues in cases without internet connectivity, on another it causes a lot of latency the first time this gets called.

**Comment by bbatsov** [comment]
Fixed by #96
